src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java

```diff
@@ -78,6 +78,24 @@ public class Mining {
 	@ConfigAccordionId(id = 0)
 	public int emissaryWaypoints = 1;
 
+	@Expose
+	@ConfigOption(
+		name = "Enable Powder Ghast Waypoints",
+		desc = "Shows a waypoint of the location of the Powder Ghast"
```

Comment:
I suggest the wording 'at the location' in these sentences. Less repeating 'of's.